flashinfer/fused_moe/core.py

```diff
@@ -456,10 +423,10 @@ def refine_tuning_config(cls, tune_max_num_tokens: int):
             )
 
     @register_custom_op(
-        "flashinfer::cutlass_fused_moe_sm100",
+        "flashinfer::cutlass_fused_moe",
         mutates_args=(""),
     )
-    def cutlass_fused_moe_sm100(
+    def cutlass_fused_moe(
```

Comment:
This is the public API right? Can we take this opportunity to write a docstring and document it all?

Reply:
Nevermind, it is the one below!